services/worker/tests/fixtures/hub.py

```diff
@@ -714,4 +722,13 @@ def hub_datasets(
             ),
             "parquet_and_info_response": None,
         },
+        "duckdb_index": {
+            "name": hub_public_duckdb_index,
+            "config_names_response": create_config_names_response(hub_public_duckdb_index),
+            "splits_response": create_splits_response(hub_public_duckdb_index),
+            "first_rows_response": create_first_rows_response(hub_public_duckdb_index, TEXT_cols, TEXT_rows),
+            "parquet_and_info_response": create_parquet_and_info_response(
+                dataset=hub_public_duckdb_index, data_type="csv"
+            ),
+        },
```

Comment:
Just FYI I refactored this fixture at https://github.com/huggingface/datasets-server/pull/1373 - so depending on which one is merged first we'll have to fix a small conflict

Reply:
Done, I created a new fixture for duckdb_index